sdcm/mgmt.py

```diff
@@ -293,6 +298,58 @@ def __init__(self, manager_node, cluster_id, ssh_identity_file=None, client_encr
         self.client_encrypt = client_encrypt
         self.ssh_identity_file = ssh_identity_file
 
+    def create_backup_task(self, location_list, retention=None, num_retries=None,  # pylint: disable=too-many-arguments,too-many-locals,too-many-branches
```

Comment:
maybe it would be easier to put the params into a dict that the keys are the actual name to pass in the command

that's a whole lot of boiler plate for one command

```python
for arg, value in params.items():
   if isinstance(value, (str,int)) and value:
        arguments.append('--{} '.format(arg) + str(value))
   if isinstance(value, bool) and value:
        arguments.append('--{} '.format(arg))
   if isinstance(value, list) and value:
        arguments.append('--{} {}'.format(arg, ','.join(value))
```